.env.sample

```
@@ -1,12 +1,12 @@
-# See https://docs.karakeep.app/configuration for more information
+# See the repository documentation for configuration details.
 DATA_DIR=<path>
 NEXTAUTH_SECRET=<secret>
 
 # Operator helper for pulling the full prod state into local development.
 # Real values belong only in .env.
 KARAKEEP_PROD_SSH_HOST=vps
 KARAKEEP_PROD_SSH_USER=
-KARAKEEP_PROD_COMPOSE_DIR=/home/praya/karakeep
+KARAKEEP_PROD_COMPOSE_DIR=/home/praya/marka
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The KARAKEEP_PROD_COMPOSE_DIR key should go before the KARAKEEP_PROD_SSH_HOST key

(UnorderedKey)